html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">
+      Email
+      <div>
+        <input id="email" type="text" name="email">
+      </div>
+    </label>
+    <label for="reason">
+      Why is coding cool?
+      <div>
+        <select id="reason" name="reason">
+          <option value="default">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </label>
+    <label for="languages">
+      Favorite Language (choose one)
+      <div>
+        <input type="radio" name="language" value="html" /> HTML
+        <input type="radio" name="language" value="css" /> CSS
+        <input type="radio" name="language" value="javascript" /> JavaScript
+      </div>
+    </label>
+    <label for="information">
+      Would you like to sign up for more information?
+      <div>
+        <input type="checkbox" name="signup" /> Sign me up!
+      </div>
+    </label>
+    <label for="comments">
+      Comments
+      <div>
+        <textarea name="comments" cols="25" rows="10">
+        </textarea>
+      </div>
+      <label for="submit">
+        <input type="submit" name="submit" value="Submit" />
+      </label>
+    </label>
```

Comment:
The labels in this section don't have an `id` on their correlated input element that matches the `for` attribute value